modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendarAccessibility.testcase

```diff
@@ -76,4 +76,38 @@ definition {
 		}
 	}
 
+	@description = "LPS-185089 Verifies that the focus is not lost after closing the 'Edit Event' modal."
+	@priority = 3
+	test FocusIsNotLostAfterClosingTheModal {
+		SetWindowSize(value1 = "1920,1540");
```

Comment:
I needed to add it so the focus goes to the right place at this resolution, even if I apply `ScrollWebElementIntoView` before the element is focused, I think the resolution is not enough, so that way works fine.

Reply:
What do you mean by the element was not focused? Was it hidden or not visible? I would expect any element should be able to be focused at standard browser desktop dimensions.

Reply:
> What do you mean by the element was not focused? Was it hidden or not visible? I would expect any element should be able to be focused at standard browser desktop dimensions.

When the modal is closed, the focus goes to the first focusable element in the page, instead of going to the `Select` button, depending on the resolution... 🤔

Reply:
**With** `SetWindowSize(value1 = "1920,1540");`

https://github.com/liferay-peds/liferay-portal/assets/63858122/98ee8fdf-de2d-4fdb-bc08-1e26dfe6332b

-------------------------------------


**Without** `SetWindowSize(value1 = "1920,1540");`

https://github.com/liferay-peds/liferay-portal/assets/63858122/d3c20d83-4140-42c2-8700-ae173beb0a45

Reply:
Can you reproduce this issue manually? If so, this is likely a bug that needs to be reported.